### DISCUSSION TITLE
Change Text Alignment by Responsive Breakpoint

### DISCUSSION THREAD

**Discussion by gmeben** [quasarframework/quasar#9395] [General - Components / Directives / etc] [answered]
Assuming the viewport is at least within a small breakpoint (600px or more) and given the following markup:

```
<div class="text-right">
    Text
</div>
```

How would I conditionally change the alignment of the text in this container upon the viewport changing to an extra small breakpoint (599px or less) so that the `text-right` helper class is no longer applied and the text is center aligned?

Put another way, I want this behavior:

| Breakpoint   | Desired Text Alignment | 
| ----------- | ------------ |
| Extra Small  |  Center |
| Small and Up  | Right |

How do I achieve this with Quasar?

**Comment by metalsadman** [comment, marked as answer]
@gmeben take a look into quasar's [Screen Plugin](https://quasar.dev/options/screen-plugin#introduction). ie.
```html
<div :class="`text-${$q.screen.lt.sm ? 'center' : 'right'}`">
  Text
</div>
```

**Comment by gmeben** [comment]
@metalsadman Thank you. I'm going to do some testing soon to verify if that suggestion meets my needs. I'm optimistic that it will.

**Comment by gmeben** [comment]
This works. Thank you again.
